src/main/java/com/microsoft/graph/authentication/AuthConstants.java

```diff
@@ -0,0 +1,14 @@
+package com.microsoft.graph.authentication;
+
+public class AuthConstants {
+
+    public static class Tenants
```

Comment:
Is there a plan to use AuthConstants.Tenants? If not, please remove this.

Reply:
This was copied directly from the Service Library, I wasn't sure if there were any changes that needed to be made so I left them as they were directly from the Service Library. If the service Library were to reference this Core library, and this class in particular, would it cause a breaking change to remove this sub-Class?

Reply:
Now that I think of it, we would want to remove this from the service library, and use this eventually.